src/frontend/components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetVOD/HarvestVOD/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { Paragraph } from 'grommet';
+import React, { Fragment, useEffect } from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { harvestLive } from 'data/sideEffects/harvestLive';
+import { useVideo } from 'data/stores/useVideo';
+import { Video } from 'types/tracks';
+import { useFetchButton } from 'components/FetchButton';
+
+const messages = defineMessages({
+  harvestLiveMessage: {
+    defaultMessage:
+      'You can harvest your live to create the vod. Only the recorded sections will be available in the vod.',
+    description:
+      'Message displays when we can harvest a live to create the vod.',
+    id: 'components.DashboardVideoLiveWidgetVOD.harvestLiveMessage',
+  },
+  harvestLiveButtonLabel: {
+    defaultMessage: 'Create vod',
+    description: 'Label of the button to harvest the live and create the vod',
+    id: 'components.DashboardVideoLiveWidgetVOD.harvestLiveButtonLabel',
+  },
+});
+
+interface HarvestVODProps {
+  video: Video;
+}
+
+export const HarvestVOD = ({ video }: HarvestVODProps) => {
+  const intl = useIntl();
+  const [state, setState, FetchButton] = useFetchButton();
+  const { updateVideo } = useVideo((storeState) => ({
+    updateVideo: storeState.addResource,
+  }));
+
+  useEffect(() => {
```

Comment:
just asking, no need to use useAsyncEffect and await apiCall()

Reply:
I dont't tink so, this is a simple fetch on button clik, if an error occured a toast with a message will be displayed

Reply:
I don't like `useAsyncEffect` since it can leaks (by calling `setState` in the async function on an unmounted component